.travis.yml

```diff
@@ -6,7 +6,6 @@ go:
   - 1.5.1
   - tip
 before_install:
-  - go get golang.org/x/tools/cmd/vet
```

Comment:
The travis build failed because that repo no longer works.  I wouldn't put it in the PR, but that's the only way it got past travis.